cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -128,6 +129,14 @@ def _create_entrance_exam(request, course_key, entrance_exam_minimum_score_pct=N
     }
     CourseMetadata.update_from_dict(metadata, course, request.user)
 
+    # Create the entrance exam section item.
+    create_xblock(
+        parent_locator=unicode(created_block.location),
+        user=request.user,
+        category='sequential',
+        display_name=_('Entrance Exam - Subsection')
```

Comment:
Seems like this still needs resolution, but something this small shouldn't prevent moving to 2nd review.